vkcs/mlplatform/resource_deploy.go

```diff
@@ -0,0 +1,542 @@
+package mlplatform
```

Comment:
lets rename the file to fit naming schema

Reply:
renamed